sections/main-collection-banner.liquid

```diff
@@ -46,7 +46,7 @@
     {
       "type": "checkbox",
       "id": "show_collection_description",
-      "default": false,
+      "default": true,
       "label": "t:sections.main-collection-banner.settings.show_collection_description.label"
     },
     {
```

Comment:
I just noticed that even if the setting is on, but the string is empty it still renders the `div`

https://screenshot.click/07-41-0tkji-x4y83.png

Should we also be checking the length of the content?

```
      {%- if section.settings.show_collection_description and  collection.description.length > 0 -%}
        <div class="collection-hero__description rte">{{ collection.description }}</div>
      {%- endif -%}
```

Reply:
That should be fine, since we have `div:empty { display: none }`.  This way, we can hide it without adding complexity to the Liquid.